static/app/components/search/sources/commandSource.tsx

```diff
@@ -94,9 +99,37 @@ if (NODE_ENV === 'development' && window?.__initialData?.isSelfHosted === false)
   });
 }
 
+async function createCopyDSNAction(api: Client, orgId: string, projectId: string) {
+  const data: ProjectKey[] = await api.requestPromise(
+    `/projects/${orgId}/${projectId}/keys/`
+  );
```

Comment:
Can we instead make the API request when we fire the action, vs making the request when opening the modal and causing the whole modal to delay opening while we wait for this.